bot.ml

```diff
@@ -786,15 +786,158 @@ let callback _conn req body =
+                                  ~pr_id:pr.id
+                                  ~commit_headline:
+                                    (f "Merge PR #%d: %s" pr.issue.number
+                                       comment_info.issue.title)
+                                  ~commit_body:
+                                    ( List.fold_left
+                                        reviews_info.approved_reviews
+                                        ~init:"Reviewed-by:\n" ~f:(fun s r ->
+                                          s ^ f "- %s\n" r)
+                                    ^
+                                    if
+                                      not
+                                        (List.is_empty
+                                           reviews_info.comment_reviews)
+                                    then
+                                      List.fold_left
+                                        reviews_info.comment_reviews
+                                        ~init:"Ack-by:\n" ~f:(fun s r ->
+                                          s ^ f "- %s\n" r)
+                                    else "" )
+                                  ~merge_method:`MERGE
+                                >>= fun () ->
+                                match
+                                  List.fold_left ~init:[] reviews_info.files
+                                    ~f:(fun acc f ->
+                                      if
+                                        string_match
+                                          ~regexp:
+                                            "dev/ci/user-overlays/\\(.*\\)" f
+                                      then Str.matched_group 1 f :: acc
+                                      else acc)
+                                with
+                                | [] ->
+                                    Lwt.return ()
+                                | overlays ->
+                                    GitHub_mutations.post_comment ~bot_info
+                                      ~message:
+                                        (f
+                                           "@%s: Please take care of the \
+                                            following overlays:\n\
+                                            %s"
+                                           comment_info.author
+                                           (List.fold_left overlays ~init:""
+                                              ~f:(fun s o -> s ^ f "- %s\n" o)))
+                                      ~id:pr.id )
+                            | Error _ ->
+                                GitHub_mutations.post_comment ~bot_info
+                                  ~message:
+                                    (f
+                                       "@%s: You can't merge this PR as you're \
+                                        not a member of the Pushers team."
```

Comment:
This can be a way to designate precisely the pushers team without spamming its members:

```suggestion
                                        not a member of the `@coq/pushers` team."
```